aws/resource_aws_ebs_snapshot_copy.go

```diff
@@ -153,10 +153,14 @@ func resourceAwsEbsSnapshotCopyDelete(d *schema.ResourceData, meta interface{})
 		}
 
 		ebsErr, ok := err.(awserr.Error)
-		if ebsErr.Code() == "SnapshotInUse" {
+		if ok && ebsErr.Code() == "SnapshotInUse" {
```

Comment:
Leaving as non-blocking feedback for next time.

```suggestion
		if isAWSErr(err, "SnapshotInUse", "") {
```

Reply:
fixed.